docs/oidc_testing/README.md

```diff
@@ -0,0 +1,81 @@
+# Bring up a cluster w/ the OIDC code enabled.
+1. Set up dev environment (which you've probably already done)
+   ```
+   git clone git@github.com:kubic-project/automation
+   cd automation
+   ./caasp-devenv --setup
+   ```
+1. check out OIDC-aware velum
+   ```
+   cd ../velum
+   git remote add dannysauer git@github.com:dannysauer/velum
+   git fetch dannysauer
+   git checkout dannysauer/oidc_support_3
+   ```
+1. check out OIDC-aware salt
+   ```
+   cd ../salt
+   git remote add dannysauer git@github.com:dannysauer/salt
+   git fetch dannysauer
+   git checkout dannysauer/oidc_support
+   ```
+1. `cd ../automation && ./caasp-devenv --build --bootstrap` (with whatever other options, like `-L provo`)
+
+# Bring up the stand-alone dex OIDC Provider (OP)
+1. `cd ../velum/docs/oidc_testing`
+1. `cp examples/config-dev.yaml{.example,}`
+1. edit `examples/config-dev.yaml`
+   * there is a "NOTE:" in each place you need to review, which will show up an alternate color in vim w/ syntax hilighting
```

Comment:
Mention of 'alternate color in vim' is overly verbose here. Mention of 'NOTE' even is also verbose. This can be dropped. Saying "modify as needed to suit your configuration" or something like that is sufficient.

Reply:
I agree with removing the vim part.  Later steps assume that only the `NOTE`-delimited sections have been modified, so suggesting only modifying those two places seemed like a good idea.  I think I can word this better, though.

Reply:
Reworded in cbed0d6